src/appkit.cs

```diff
@@ -3688,6 +3761,15 @@ public partial interface NSColor : NSCoding, NSCopying, NSSecureCoding, NSPasteb
 		[Static, Export ("quaternaryLabelColor")]
 		NSColor QuaternaryLabelColor { get; }
 		
+		[Mac (10,12)]
+		[Static]
+		[Export ("colorWithDisplayP3Red:green:blue:alpha:")]
+		NSColor ColorWithDisplayP3Red (nfloat red, nfloat green, nfloat blue, nfloat alpha);
```

Comment:
`Red` should not be part of the method name. A prefix (verb) would be nice, check `uikit.cs` to use the same name.